portal/frontend/src/main/kotlin/com/sourceplusplus/portal/page/TracesPage.kt

```diff
@@ -1,28 +1,91 @@
 package com.sourceplusplus.portal.page
 
+import com.sourceplusplus.portal.extensions.eb
+import com.sourceplusplus.portal.extensions.jq
+import com.sourceplusplus.portal.extensions.moment
 import com.sourceplusplus.portal.template.*
+import com.sourceplusplus.protocol.ProtocolAddress.Global.Companion.ClickedDisplaySpanInfo
+import com.sourceplusplus.protocol.ProtocolAddress.Global.Companion.ClickedDisplayTraceStack
+import com.sourceplusplus.protocol.ProtocolAddress.Global.Companion.ClickedDisplayTraces
+import com.sourceplusplus.protocol.ProtocolAddress.Global.Companion.TracesTabOpened
+import com.sourceplusplus.protocol.ProtocolAddress.Portal.Companion.DisplayInnerTraceStack
+import com.sourceplusplus.protocol.ProtocolAddress.Portal.Companion.DisplaySpanInfo
+import com.sourceplusplus.protocol.ProtocolAddress.Portal.Companion.DisplayTraceStack
+import com.sourceplusplus.protocol.ProtocolAddress.Portal.Companion.DisplayTraces
 import com.sourceplusplus.protocol.artifact.trace.TraceOrderType
 import com.sourceplusplus.protocol.artifact.trace.TraceOrderType.*
+import com.sourceplusplus.protocol.artifact.trace.TraceResult
+import com.sourceplusplus.protocol.artifact.trace.TraceSpanInfo
 import com.sourceplusplus.protocol.artifact.trace.TraceSpanInfoType.END_TIME
 import com.sourceplusplus.protocol.artifact.trace.TraceSpanInfoType.START_TIME
 import com.sourceplusplus.protocol.artifact.trace.TraceStackHeaderType.TIME_OCCURRED
 import com.sourceplusplus.protocol.artifact.trace.TraceStackHeaderType.TRACE_ID
 import com.sourceplusplus.protocol.artifact.trace.TraceTableType.*
 import com.sourceplusplus.protocol.portal.PageType.*
 import kotlinx.browser.document
+import kotlinx.browser.window
+import kotlinx.html.*
 import kotlinx.html.dom.append
+import kotlinx.html.dom.create
+import kotlinx.html.js.onClickFunction
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromDynamic
 import org.w3c.dom.Element
+import org.w3c.dom.HTMLElement
+import org.w3c.dom.HTMLTableRowElement
+import org.w3c.dom.get
+import kotlin.js.json
+import kotlin.math.round
 
 /**
  * todo: description.
  *
  * @since 0.0.1
  * @author [Brandon Fergerson](mailto:bfergerson@apache.org)
  */
+@ExperimentalSerializationApi
 class TracesPage(
     private val portalUuid: String,
     private val traceOrderType: TraceOrderType
 ) {
+    init {
+        console.log("Traces tab started")
+        setupUI()
+        eb.onopen = {
+            js("portalConnected()")
+            eb.registerHandler(DisplayTraces(portalUuid)) { error: String, message: dynamic ->
+                console.log("########## Calling displayTraces...")
+                val body: dynamic = message.body
+                val traceResult: TraceResult = Json.decodeFromDynamic(body)
+                console.log(">>>>>>>>>> artifactQualifiedName = ${traceResult.artifactQualifiedName}")
```

Comment:
You have these ">>>>>" console loggings everywhere. Please remove them when you commit/create PRs. They should only be for you to debug.